_translations/ja.yml

```diff
@@ -495,7 +495,7 @@ ja:
     contributorsorder: "（コミット数順）"
     devcommunities: "開発者コミュニティ"
     devcommunitiesintro: "以下のチャットルームとウェブサイトでは、ビットコイン開発に関するディスカッションを主催しています。投稿される前に必ず規約をお読みください。"
-    ircjoin: "フリーノード上の<a href=\"https://webchat.freenode.net/?channels=bitcoin-core-dev\">IRCチャンネル #bitcoin-core-dev</a>。"
+    ircjoin: "フリーノード上の<a href=\"https://web.libera.chat/#bitcoin-core-dev\">IRCチャンネル #bitcoin-core-dev</a>。"
```

Comment:
Please modify to next sentence because we don't use freenode hereafter. Other language's files may have same issue.
"LiberaChat上の<a href=\"https://web.libera.chat/#bitcoin-core-dev\">IRCチャンネル #bitcoin-core-dev</a>。"